packages/vue3/src/components/Modal/Modal.vue

```diff
@@ -11,9 +11,16 @@ const props = withDefaults(defineProps<ModalProps>(), {
 });
 
 const emit = defineEmits(['update:visible', 'onConfirm']);
+
+const slots = defineSlots<{
```

Comment:
Since we're just using slots in the template, we can omit this line and use the [$slots](https://vuejs.org/guide/components/slots#conditional-slots) directly there.

Reply:
Fixed [4423478](https://github.com/youcan-shop/youcan-ui/pull/427/commits/4423478828714e8edcf968d7d5a5f2baf9c0c94a)